src/main/java/org/isf/admission/rest/AdmissionController.java

```diff
@@ -212,73 +197,72 @@ public ResponseEntity<List<AdmittedPatientDTO>> allAdmittedPatients()
+					@RequestParam(name = "dischargerange", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime[] dischargeRange)
+					throws OHServiceException {
 		LOGGER.info("Get admitted patients search terms: {}", searchTerms);
 		if (admissionRange != null && admissionRange.length == 2) {
 			LOGGER.debug("Get admissions started between {} and {}", admissionRange[0], admissionRange[1]);
 		}
 		if (dischargeRange != null && dischargeRange.length == 2) {
-			LOGGER.debug("Get admissions ended between {} and {}", dischargeRange[0], dischargeRange[1]);
+			LOGGER.debug("Get admissions that end between {} and {}", dischargeRange[0], dischargeRange[1]);
 		}
-		
+
 		List<AdmittedPatient> admittedPatients = admissionManager.getAdmittedPatients(admissionRange, dischargeRange, searchTerms);
 		if (admittedPatients.isEmpty()) {
 			return ResponseEntity.status(HttpStatus.NO_CONTENT).body(null);
 		}
-		
+
 		return ResponseEntity.ok(admittedMapper.map2DTOList(admittedPatients));
 	}
-	
+
 	/**
-	 * Get all the {@link Admission}s for the specified {@link Patient} code.
+	 * Get all the {@link Admission}s that start in the specified range
 	 * 
-	 * @param patientCode
+	 * @param admissionRange
 	 * @return the {@link List} of found {@link Admission} or NO_CONTENT otherwise.
 	 * @throws OHServiceException
 	 */
 	@GetMapping(value = "/admissions", produces = MediaType.APPLICATION_JSON_VALUE)
 	public ResponseEntity<List<AdmissionDTO>> getAdmissions(
-			@RequestParam(name = "patientcode", defaultValue = "0", required = false) int patientcode,
-			@RequestParam(name = "admissionrange", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime[] admissionRange,
-			@RequestParam(name = "dischargerange", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime[] dischargeRange)
-			throws OHServiceException {
-		LOGGER.info("Get admissions of patients by id: {}", patientcode);
-		if (admissionRange != null && admissionRange.length == 2) {
-			LOGGER.debug("Get admissions started between {} and {}", admissionRange[0], admissionRange[1]);
-		}
-		if (dischargeRange != null && dischargeRange.length == 2) {
-			LOGGER.debug("Get admissions ended between {} and {}", dischargeRange[0], dischargeRange[1]);
-		}
-		
-		List<AdmittedPatient> admittedPatients = new ArrayList<AdmittedPatient>();
-		if (patientcode == 0) {
-			admittedPatients = admissionManager.getAdmittedPatients(admissionRange, dischargeRange, "");
-		} else {
-			admittedPatients = admissionManager.getAdmittedPatients(admissionRange, dischargeRange, Integer.toString(patientcode));
-		}
-		if (admittedPatients.isEmpty()) {
+					@RequestParam(name = "admissionrange") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime[] admissionRange,
+					@RequestParam(value = "page", required = false, defaultValue = "0") Integer page,
```

Comment:
I'm seeing the use of `Integer` in many of these methods; why aren't they just simple `int`?